lib/msal-node/src/request/ManagedIdentityRequestParams.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * ManagedIdentityRequest
+ * - forceRefresh - forces managed identity requests to skip the cache and make network calls if true
+ * - resource  - resource requested to access the protected API. It should be of the form "{ResourceIdUri}" or {ResourceIdUri/.default}. For instance https://management.azure.net or, for Microsoft Graph, https://graph.microsoft.com/.default
+ */
+export type ManagedIdentityRequestParams = {
```

Comment:
I see your confusion. However, ManagedIdentityRequest only exists so that caching functionality from ClientCredentials can be re-used by ManagedIdentity. This functionality requires an authority, correlationId, etc. The  developer can't pass in this information, they can only pass in forceRefresh and resource, which are in ManagedIdentityRequestParameters.